local/modules/HookAnalytics/HookAnalytics.php

```diff
@@ -12,9 +12,21 @@
 
 namespace HookAnalytics;
 
+use Propel\Runtime\Connection\ConnectionInterface;
+use Thelia\Model\ConfigQuery;
+use Thelia\Model\LangQuery;
 use Thelia\Module\BaseModule;
 
 class HookAnalytics extends BaseModule
 {
-
+    public function update($currentVersion, $newVersion, ConnectionInterface $con = null)
+    {
+        if (version_compare($newVersion, $currentVersion, ">")){
```

Comment:
This will copy the trackingcode at each update even for all future updates and erase filed values. You should check that the current version is less or equal to 2.4.3 and the new version greater than 2.4.3